applications/matter_bridge/CMakeLists.txt

```diff
@@ -10,15 +10,6 @@ cmake_minimum_required(VERSION 3.20.0)
 set(mcuboot_KCONFIG_ROOT "\\\${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/chip-module/Kconfig.mcuboot.root")
 set(hci_ipc_KCONFIG_ROOT "\\\${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/chip-module/Kconfig.hci_ipc.root")
```

Comment:
There was a meeting a long time ago about fixing this up, I can't remember what the solution for this was going to be, is that going to be done in a later commit?

Reply:
Yes, I'm aware of that. We're going to manage that in a separate PR because there are already plenty of changes.